scripts/build-local.js

```diff
@@ -0,0 +1,30 @@
+var path = require('path')
+var fs = require('fs')
+var spawn = require('child_process').spawn
+
+// Runs `npm run build-local` if the lib dir is absent
```

Comment:
This script does more than what the comment says. I think it should be something like, "If installed as an unreleased git dependency, installs the dev dependencies and compiles the source"